docs/ARDUPILOT.md

```diff
@@ -0,0 +1,55 @@
+# mLRS Documentation: ArduPilot Setup #
+
+([back to main page](../README.md))
+
+([back to CRSF page](CRSF.md))
```

Comment:
I would remove, since one may have come to here also from e.g. the mBridge page (if the info on this page really would only be for CRSF when it alls should go to CRSF, or be a subpage of CRSF). IMHO :)

Reply:
CRSF page link removed.